packages/canary-web-components/src/components/ic-data-table/ic-data-table.stories.mdx

```diff
@@ -1583,9 +1583,9 @@ To set the min and max width of a table cell, set the `table-layout` attribute t
 </script>
 ```
 
-### Action Element
+### Action Element and Action OnClick
 
-The example below demonstrates action elements appearing in table cells after being passed in via the data.
+The example below demonstrates action elements and Action OnClicks appearing in table cells after being passed in via the data. When dealing with Large sets of data, its easier to pass in the Elements and Onclick function and keeping the accessibility intact.
```

Comment:
similar to react storybook. please just use the same text as there and maybe don't worry about the large sets of data & accessibility sentence